docs/config.json

```diff
@@ -730,6 +730,11 @@
       "destination": "/machine-workload-identity/workload-identity/workload-attestation/",
       "permanent": true
     },
+    {
+      "source": "/machine-workload-identity/workload-identity/workload-attestation/",
```

Comment:
Was this page removed in a previous change?

Reply:
8545b40fdd